### ISSUE TITLE
Embed it in existing form with other inputs

### ISSUE THREAD

**Issue by harrysrevis** [danielm/uploader#83]
Hello, 
i'd like to ask if it's possible to embed your script in existing form and pass the data through php controller.
Something like codeigniter.

Thanks!

**Comment by SapientHetero** [comment]
Absolutely. You can start by embedding the HTML from one of the examples in your web page and loading the associated javascript and css files. I recommend combining the .js and .css files so you only load one of each.

**Comment by harrysrevis** [comment]
@SapientHetero 
Can i pass other input data too (and there are many of them :P )? Since i'm trying to do the "form.serialize()" function and images are not uploaded. 
Kinda new in jquery so i might do something wrong.
